### PR TITLE
Output md

### PR DESCRIPTION
**Pull request by ramprasadn**
## PR checklist
Update Output.md to describe output files described by default.

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR> -stub`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test_one_sample,docker --outdir <OUTDIR> -stub`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by jemten — COMMENTED** [review]
some suggestions

File: docs/output.md [hunk 1]
```diff
@@ -6,38 +6,192 @@ This document describes the output produced by the pipeline. Most of the plots a
...
+
+- `{outputdir}/alignment/`
+  - `*.bam`: Bam file containing report containing quality metrics.
+  - `*.bai`: Zip archive containing the FastQC report, tab-delimited data file and plot images.
+  - `*.txt`: Text file containing the dedup metrics.
+  </details>
+
+##### Sentieon dedup
+
+[Sentieon dedup](https://support.sentieon.com/manual/DNAseq_usage/dnaseq/#remove-or-mark-duplicates) is the algorithm used by Sentieon's driver to remove duplicate reads. Only reads aligned by Sentieon's implementation of bwa are processed by this algorithm.
```
[the remark this answers is not in the document]
Comment (ramprasadn) [reply docs/output.md]: Sentieon doesn't do that it by default, but we use a specific parameter to remove the duplicates.

File: docs/output.md [hunk 2]
```diff
@@ -6,38 +6,192 @@ This document describes the output produced by the pipeline. Most of the plots a
...
+
+##### Sentieon
+
+[Sentieon's bwa mem](https://support.sentieon.com/manual/DNAseq_usage/dnaseq/#map-reads-to-reference) is the software accelerated version of the bwa-mem algorithm. It is used to efficiently perform the alignment using BWA. Aligned reads are then coordinate sorted using Sentieon's [sort](https://support.sentieon.com/manual/usages/general/#util-syntax) utility. These files are treated as intermediates and are not placed in the output folder by default. It is not the default aligner, but it can be chosen over bwamem2 by setting `--aligner` option to sentieon.
+
+#### Duplicate marking
+
+##### Picard's MarkDuplicates
+
+[Picard MarkDuplicates](https://broadinstitute.github.io/picard/command-line-overview.html#MarkDuplicates) is used for the removal of PCR duplicates that can occur during library amplification. This is essential as the presence of such duplicates results in false inflated coverages, which in turn can lead to overly-confident genotyping calls during variant calling. Only reads aligned by Bwa-mem2 are processed by this tool.
```
[the remark this answers is not in the document]
Comment (ramprasadn) [reply docs/output.md:73]: Good catch 😄

File: docs/output.md [hunk 3]
```diff
@@ -6,38 +6,192 @@ This document describes the output produced by the pipeline. Most of the plots a
...
+  - [SVDB query](#svdb-query)
+  - [VEP](#vep-1)
+- [Mitochondrial analysis](#mitochondrial-analysis)
+  - [Alignment and variant calling](#alignment-and-variant-calling)
+  - [Annotation:](#annotation-)
+    - [Haplogrep2](#haplogrep2)
+    - [VCFanno](#vcfanno-1)
+    - [VEP](#vep-2)
+- [Rank variants and filtering](#rank-variants-and-filtering)
+  - [GENMOD](#genmod)
```
[the remark this answers is not in the document]
Comment (ramprasadn) [reply docs/output.md:54]: I have added VEP filter now 👍🏻

File: README.md
[the remark this answers is not in the document]
Comment (ramprasadn) [reply README.md:1]: Done 👍🏻

File: docs/output.md [hunk 4]
```diff
@@ -6,38 +6,192 @@ This document describes the output produced by the pipeline. Most of the plots a
...
+- [Variant calling - repeat expansions](#variant-calling---repeat-expansions)
+  - [ExpansionsHunter](#expansionshunter)
+  - [stranger](#stranger)
+- [Annotation - SNV](#annotation---snv)
+  - [bcftools roh](#bcftools-roh)
+  - [VCFanno](#vcfanno)
+  - [VEP](#vep)
+- [Annotation - SV](#annotation---sv)
+  - [SVDB query](#svdb-query)
+  - [VEP](#vep-1)
```
[the remark this answers is not in the document]
Comment (ramprasadn) [reply docs/output.md:46]: Fixed it 👍🏻

File: docs/output.md [hunk 5]
```diff
@@ -6,38 +6,192 @@ This document describes the output produced by the pipeline. Most of the plots a
...
+    - [FastQC](#fastqc)
+    - [Mosdepth](#mosdepth)
+    - [Picardtools](#picardtools)
+    - [Qualimap](#qualimap)
+    - [Sention WgsMetricsAlgo](#sention-wgsmetricsalgo)
+    - [TIDDIT's cov and UCSC WigToBigWig](#tiddit-s-cov-and-ucsc-wigtobigwig)
+  - [Reporting](#reporting)
+    - [MultiQC](#multiqc)
+- [Variant calling - SNV](#variant-calling---snv)
+  - [DeepVariant](#deepvariant)
```
[the remark this answers is not in the document]
Comment (ramprasadn) [reply docs/output.md:31]: I think GLnexus is specific.. I have added the link to GLnexus in the DeepVariant section now. What do you think?

File: docs/output.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jemten) [reply docs/output.md]: In a coming PR we should harmonise this so that markduplicates and sentieon either mark or remove depending on parameter input.

**Review by jemten — APPROVED** [review]
Looks good 👍

File: docs/output.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ramprasadn) [reply docs/output.md]: I agree! 👍🏻

**Pull request merged**